torch/csrc/jit/passes/onnx/remove_inplace_ops_for_onnx.cpp

```diff
@@ -531,7 +534,7 @@ std::deque<std::string> findSubModuleAttr(
       moduleNames.push_front(node->s(attr::name));
       node = node->inputs()[0]->node();
     } else {
-      return moduleNames;
+      break;
```

Comment:
is this change intended? ..

Reply:
Yes, since attrModule should still be updated and assigned the inner module.